scripts/heartcheck

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
```

Comment:
What about mentioning this script in README?

Reply:
Nice one! I've added it